Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -400,11 +406,13 @@ - (void)recordError:(NSError *)error {
 }
 
 - (void)recordError:(NSError *)error userInfo:(NSDictionary<NSString *, id> *)userInfo {
-  FIRCLSUserLoggingRecordError(error, userInfo);
+  NSString *rolloutsInfo = [_remoteConfigManager getRolloutAssignmentsEncodedJsonString];
+  FIRCLSUserLoggingRecordError(error, userInfo, rolloutsInfo);
```

Comment:
Could we suffix this variable name with -JSON? eg. rolloutsInfoJSON? (here and in the function definition).

Just as a self-documenting variable since it's just a string